Start scratch of be_kind.md (#10)
nikomatsakis/rustacean-principles: src/how_to_rustacean/be_kind.md

# 💖 Be kind and considerate

> You don't have to agree with everyone, but you should always be kind and considerate.

Pushing Rust forward is already a tough task, being kind and avoiding conflicts prevents us from making it even harder.

> Rust is being built and used by different people from around the globe, everyone got a personal opinion when it comes to what is the best for the language, so disagreements are inevitable.

Reviewer: ```suggestion
Rust is being built and used by different people from around the globe and everyone has their own opinion on what is the best for the language, so disagreements are inevitable.
```

Author: Updated :+1:.
